modules/config/home-cursor.nix

```diff
@@ -0,0 +1,149 @@
+{ config, options, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  inherit (config.home) stateVersion;
+
```

Comment:
```suggestion
```

This isn't referenced.

Reply:
I forgot to remove this after some experimentating with backward compatibility. Thanks!